configs/nri-prometheus-config.yml.sample

```diff
@@ -19,6 +18,9 @@ integrations:
       # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
       # scrape_timeout: "30s"
 
+      # Length in time to distribute the scraping from the endpoints.
+      scrape_duration: "5s"
```

Comment:
I think this two configs will have duplicated functionality with `interval` and `timeout` from the Agent and in the future we should find a way to just keep the Agent ones

Reply:
yes. What I'm not 100% sure yet is if we are going to keep the "standalone" behaviour and so we would need these options, of if we are completely replacing the way POMI works to only work with the agent